utils/core.php

```diff
@@ -92,3 +96,21 @@ function restrict_access(...$permissions)
         inject_in_template();
     }
 }
+
+/**
+ * Format a date.
+ * @param string|int|DateTime $date The date either as a string (2022-12-25), a timestamp, or a DateTime
+ * @return formatted_date A string in the format: 25 Déc 2022
+ */
+function format_date($date)
+{
+    //return formatter()->format(date_create($date));
```

Comment:
This is the way to go if you wanna have localized formatting. Feel free to edit this in order to deal with different types like `DateTime`.

Reply:
Also, the $FORMAT variable is not needed anymore